docs/installation/installation.md

```diff
@@ -230,7 +230,7 @@ Build with zStandard compression support (default if detected)
 #### SQL Driver Options
 
 SQL drivers are typically used for authentication, and they may be
-used as a lib-dict backend too, which can be used by plugins for
+used as a lib-dict driver too, which can be used by plugins for
```

Comment:
The `docs/installation/upgrade/include/2.3-to-2.4.inc` page has a reference to `[[link,quota_backend_count]]` that also needs renaming to `[[link,quota_driver_count]]`.